pom.xml

```diff
@@ -3,46 +3,45 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.9</version>
+    <version>2.36</version>
   </parent>
 
-  <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>authorize-project</artifactId>
   <version>1.3.1-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Authorize Project</name>
   <description>Configure projects to run with specified authorization.</description>
 
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Authorize+Project+plugin</url>
-  
```

Comment:
Please avoid whitespace changes in the code you do not modify. IDEA can be configured in such way AFAIK. 
Such changes may cause undesired merge conflicts with other pull requests.
There is no other pending pull requests in this plugin, and it may be acceptable in this case (in a separate commit)